Updating SIP Editors & Steering Committe list + removed 1 Tech CAB member + add 3 new members

Hero-Gamer opened the pull request:
- updated files, wanna put it in SIP repo. 
- Noob at this, so please lmk if there is any issue. 
- Some stuff like In Scope or Out of Scope and By Laws haven't finalized yet (if need be), but the contact detail & bio are all done. :)
- Some email contact detail are not shown as per request by the contributor. Thought it makes more sense to allow better access to contribution without hard requiring contributor email, some people prefer to stay more anon.

Thanks to @dantrevino for teaching me how to do all the tasks!

Hero-Gamer commented:
gentle push. 
could be reviewed along with other CABs Minutes for SIP-022?

whoabuddy commented on sip editor committee/editors_list.md:
```diff
@@ -0,0 +1,127 @@
+# Consideration for SIP Editor Committee
```
Could simplify this and drop Committee:

```suggestion
# Consideration for SIP Editors
```

whoabuddy commented on sip editor committee/editors_list.md:
```diff
@@ -0,0 +1,127 @@
+
+**AcrossFire**: AcrossFire has experience as a DevOps engineer, Linux enthusiast, and a passionate Bitcoiner. He has experience with systems administration and enjoys automating maintenance tasks using tools like Puppet, Ansible, Docker, Kubernetes and others. He likes to bring this experience to Bitcoin by running Bitcoin and Stacks nodes and doing what he can to optimize the experience.
+
+**Algorithm**: Currently serving at Secret Key Lab/ Xverse.
+Advocating for Stacks tech to fully inherit Bitcoin hash power where an individual can have full ownership of their identity.
```
Is this line break intentional?

```suggestion
**Algorithm**: Currently serving at Secret Key Lab/ Xverse. Advocating for Stacks tech to fully inherit Bitcoin hash power where an individual can have full ownership of their identity.
```

whoabuddy commented on sip editor committee/editors_list.md:
```diff
@@ -0,0 +1,127 @@
+
+**Kaylee Fort**: Kayla, also known as Kaylee, serves as the Community Growth Lead at Xverse, she is also a Bitcoin educator and content creator, where she contributes to the Bitcoin ecosystem. While previously specializing in Medical Simulation at Johns Hopkins University, she has been actively involved in Bitcoin education and participating in Stacks community events. Kaylee spent over 5 years a ...
+
+**Marshall**: Marshall earns his living as a warehouse associate and forklift driver. He spends his spare time in the evenings and weekends enveloped in the Stacks ecosystem, tinkering with his nodes and talking shop with Pomeranian.btc.
+Marshall fell in love with Stacks when he learned that the mission from the beginning (even before the 2.0 launch) was to build a decentralized internet where users take back control from Web2 giants like Facebook and Google. These companies shouldn’t be allowed to say whether or not they “want” to be evil, as Google’s infamous tagline was “Don’t be evil.” Rather, the tagline should be “Can’t be  ...
+Marshall’s top core skills include his curiosity, problem solving / critical thinking, and written communication. He joined the Stacks ecosystem in February 2021 and never looked back. He graduated from Clarity Cohort 5 in 2022, he’s now excited to begin contributing to the SIP process, and he hopes to eventually ditch his forklift to contribute to the ecosystem full-time.
```
Same here, could bring all into one paragraph like the others

```suggestion
**Marshall**: Marshall earns his living as a warehouse associate and forklift driver. He spends his spare time in the evenings and weekends enveloped in the Stacks ecosystem, tinkering with his nodes and talking shop with Pomeranian.btc. Marshall fell in love with Stacks when he learned that the mission from the beginning (even before the 2.0 launch) was to build a decentralized internet where users take back control from Web2 giants like Facebook and Google. These companies shouldn’t be allowed to say whether or not they “want” to be evil, as Google’s infamous tagline was “Don’t be evil.” Rather, the tagline should be “Can’t be evil,” as Muneeb so aptly put it. Marshall’s top core skills include his curiosity, problem solving / critical thinking, and written communication. He joined the Stacks ecosystem in February 2021 and never looked back. He graduated from Clarity Cohort 5 in 2022, he’s now excited to begin contributing to the SIP process, and he hopes to eventually ditch his forklift to contribute to the ecosystem full-time.
```

whoabuddy commented on sip editor committee/editors_list.md:
```diff
@@ -0,0 +1,127 @@
+
+**Sleepi**: I am a study coordinator in the medical/pharmacy research with a strong passion for cryptocurrency, having become fascinated with its dynamic nature through day trading. As a community representative, I am committed to educating and supporting others and ensuring fair representation for all members. While valuing privacy, I continue to expand my knowledge in this rapidly-evolving indu ...
+
+**Tigs / Oli**: tigs / Oli: Yo! The name is tigs and have been a multiplayer pro gamer since the early 90s. Not quite so pro anymore but through this exposure I taught myself Cisco (CCNA) and server administration at a young age. I haven't passed high school (got a GED), have attended college for gaming dev but never completed and have been involved in a few jobs in regards to teaching, inspiring ...
```
Remove duplicate text

```suggestion
**Tigs / Oli**: Yo! The name is tigs and have been a multiplayer pro gamer since the early 90s. Not quite so pro anymore but through this exposure I taught myself Cisco (CCNA) and server administration at a young age. I haven't passed high school (got a GED), have attended college for gaming dev but never completed and have been involved in a few jobs in regards to teaching, inspiring and getting aspiring tech people into the industry as whole. Call me a Jack of all trades. Master of none! I am deeply involved with Horticulture as well (green fingers). I run several bitcoin+lightning nodes, a few miners and am deeply interested in Bitcoin as a saver of humanity, not purely as an investment vehicle. I am also involved with TSOBTC (The School of BTC).
```

whoabuddy commented on sip editor committee/editors_list.md:
```diff
@@ -0,0 +1,127 @@
+
+## About this SIP Editor Committee
+
+The role of the SIP Editors is to identify SIPs in the Draft status that can be transitioned to Accepted status. A SIP editor must be able to vet a SIP to ensure that it is well-formed, that it follows the ratification workflow faithfully, and that it does not overlap with any already-Accepted SIPs or SIPs that have since become Recommended or Ratified.
```
Some small grammar changes

```suggestion
The role of a SIP Editor is to identify SIPs in the Draft status and help transition them to Accepted status. A SIP editor must be able to vet a SIP to ensure that it is well-formed, that it follows the ratification workflow faithfully, and that it does not overlap with any already-Accepted SIPs or SIPs that have since become Recommended or Ratified.
```

whoabuddy commented on sip editor committee/editors_list.md:
```diff
@@ -0,0 +1,127 @@
+
+---
+
+_If this SIP Editor Committee interests you and you wish to join the effort, please reach out to @jennymith and @Hero-Gamer or Steering Committee members @jcnelson and @GinaAbrams_
```
Inline with the title change

```suggestion
_If becoming a SIP Editor interests you and you wish to join the effort, please reach out to @jennymith and @Hero-Gamer or Steering Committee members @jcnelson and @GinaAbrams_
```

whoabuddy commented on steering committee/members_list.md:
```diff
@@ -0,0 +1,58 @@
+
+## About this Steering Committee
+
+The Steering Committee's overarching duty is to oversee the evolution of the Stacks blockchain’s design, operation, and governance, in a way that is technically sound and feasible, according to the rules and procedures described in this document. The SC shall be guided by and held accountable by the greater community of users, and shall make all decisions with the advice of the relevant Considera ...
```
Define the abbreviation before it's used

```suggestion
The overarching duty of the Steering Committee (SC) is to oversee the evolution of the Stacks blockchain’s design, operation, and governance, in a way that is technically sound and feasible, according to the rules and procedures described in this document. The SC shall be guided by and held accountable by the greater community of users, and shall make all decisions with the advice of the relevant Consideration Advisory Boards.
```

whoabuddy reviewed (changes requested):
Left some comments, mainly small editorial stuff and feel free to push back.

For the file names, rather than create a folder for each one, you could use a descriptive file name and use the same directory:

```
sips
|- considerations
|-- (cab files)
|-- (template/readme)
|-- sip-editors.md
|-- steering-committee.md
```

If we need to separate things futher, we could put `cab-` on the front of each cab document so they sort together, but for now I think using just the file name would be clearer.

Hero-Gamer commented:
> Left some comments, mainly small editorial stuff and feel free to push back.
> 
> For the file names, rather than create a folder for each one, you could use a descriptive file name and use the same directory:
> 
> ```
> sips
> |- considerations
> |-- (cab files)
> |-- (template/readme)
> |-- sip-editors.md
> |-- steering-committee.md
> ```
> 
> If we need to separate things futher, we could put `cab-` on the front of each cab document so they sort together, but for now I think using just the file name would be clearer.

1. I've followed your advice and moved both SC and SE files into 'considerations' folder.
2. Merged all edits.
3. Removed some inactive SIP Editors from the list.

Ready to merge into Stacksgov main?

Hero-Gamer commented:
@whoabuddy can you review this too along with others, just to tidy the paperwork up to reflect the latest CAB members?

obycode reviewed (approved):
LGTM!

whoabuddy reviewed (approved):
LGTM - but we should open an issue or make sure to track/schedule getting the "To be updated." bits finished in both files.
